packaging/win/delugewin.spec

```diff
@@ -107,6 +107,7 @@ datas += copy_metadata('twisted', recursive=True)
 # Copy UI/Plugin and translation files to where pyinstaller expects
 package_data = collect_data_files('deluge')
 datas += package_data
+datas += [('../../deluge/ui/console', 'deluge/ui/console')]
```

Comment:
Adding a quick comment above this would be nice

    # Workaround for Console UI files not being included in package data